code/game/gamemodes/wizard/soulstone.dm

```diff
@@ -280,7 +280,10 @@
 				else
 					if(T.client == null)
 						to_chat(user, "<span class='userdanger'>Capture failed!</span> The soul has already fled its mortal frame. You attempt to bring it back...")
-						get_cult_ghost(T, user)
+						if(T.ghost_can_reenter())
+							get_cult_ghost(T, user)
+						else
+							get_cult_ghost(T, user, TRUE)
```

Comment:
```suggestion
						get_cult_ghost(T, user, get_new_player = !T.ghost_can_reenter())
```

Named argument is here for sake of clarity, but you may omit it if you disagree

Reply:
That works